2023/report.md

```diff
@@ -0,0 +1,155 @@
+3. Provide multidisciplinary technical and strategic support
+4. Scale and become self-sustainable, with providing best practices and advice
+5. Strong and future proof ecosystem for long-term sustainability
+
+Generally, the low-threshold funding mechanism seems to work.
+Currently there are more than 150 projects supported through NGI0 that have been packaged since 2021.
+Many of them have been abandoned, but some are very successful and actively developed.
+
+Summer of Nix in a sense has become a litmus test for project maturity:
+Participants have repeatedly taken software for a ride when trying to ensure that it builds and runs repeatably with Nix.
+This requires stable interfaces, useful documentation, and responsive maintainers.
+Participants come from diverse technical backgrounds, and their attention and critical inquiries are helpful feedback for project authors.
+We should continue in that fashion, and in 2024 focus on showcasing exceptional open source software and ensure there are up-to-date, readily available Nix packages or service modules that can be put to use with minimal effort.
+This should also involve expanding consumer-facing documentation with guides and examples.
+
+What still seems unanswered is the question of sustainability.
+This is not a technical problem, but rather a social and economic one:
+Nix infrastructure is designed for ensuring long-term availability of packages, and the Nix ecosystem already has almost all the tooling needed for maintaining sophisticated setups.
+Primarily lacking are comprehensive reference documentation, a corpus of best practices or recipes for common use cases, and more structured approaches to teaching the underlying principles.
+Continuing with Summer of Nix, NGIpkgs should eventually complement Nixpkgs as a vital place of exchange and as a curated repository of useful open source software.
+What is unclear at the moment is how to enable continued maintenance of NGIpkgs, and the effort required to effectively adopt Nix tooling into development processes is related to that.
+With Summer of Nix we successfully manage to onboard participants quickly into the Nix ecosystem, and embedding them into the community keeps growing our network of experts.
+And while Nix packaging reduces the cost of adoption of and contributions to software, project authors tend to keep relying on external support to evolve their Nix setups.
+Apart from maturing the NGIpkgs software distribution, in the next years we should therefore focus on expanding the knowledge sharing to project authors and enable them to maintain their own packages.
+Ideally participants should also, based on practical experience, contribute documentation and insights back to the Nix ecosystem to make future users and contributors less dependent on availability of individuals.
+
+A possible set of objectives in the next years, apart from concrete deliverables, could therefore be:
+
+- 2024: Select and showcase highlight projects
```

Comment:
A project being selected and packaged is highlight already?

Any additional activity is at the expense of others. But I may be misevaluating.

Reply:
Right, but literally just packaging it is what happened so far, and all it means is that some new code is buried somewhere. The minimum we can do is provide a slightly more appealing, structured overview of these packages and what they offer or enable you to do. Even better would be talking in public about why we packaged a particular project. Even in this report we're just naming them (mainly due to time constraints), and readers have to click through to some other page to see what they're about, and read some other material why we chose them.